packages/design-system/src/styles/styles.ts

```diff
@@ -0,0 +1,136 @@
+// Recidiviz - a data platform for criminal justice reform
+// Copyright (C) 2020 Recidiviz, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+// =============================================================================
+import { rgb, rgba } from "polished";
+
+const slate = "rgb(53, 83, 98)";
+const white = rgb(255, 255, 255);
+
+const basePalette = {
```

Comment:
Since we're bumping to 1.0.0, how would you feel about pulling out the shadow constants (or at minimum the colors) into here?

Reply:
I'm a little hesitant to include it at this stage as all of our shadows are unique from eachother! I'd prefer to spend some time going over our shadow palette w/ Juan and Sonia at a later time.